src/models/ontology_mapper_lm.py

```diff
@@ -0,0 +1,162 @@
+
+
+
+
+from sentence_transformers import util
+import pandas as pd
+import numpy as np
+from sentence_transformers import SentenceTransformer
+from transformers import AutoTokenizer, AutoModel, AutoModelForMaskedLM
+import torch
+import src.models.ontology_models as otm
+import src.CustomLogger.custom_logger
+import pandas as pd
+import numpy as np
+import seaborn as sns
+from io import BytesIO
+import os
+from importlib import reload
+
+logger = src.CustomLogger.custom_logger.CustomLogger()
+
+## type annotations for all the arguments and return values
+## build a different class for each type of models - sentence transformer class, LM class and LLM class
+## 3 base classes for 3 different types of models  
+## Avoid building 3 different mappers for treatment, bodysite and disease and build one curaMap 
+
+class OntoMapLM(otm.OntoModelsBase):
```

Comment:
If `Onto` stands for `Ontology`, I would just use the word `Ontology` for clarity